web/client/test-resources/csw/getRecordsResponse-gs-us_states.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?><csw:GetRecordsResponse xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" xmlns:rim="urn:oasis:names:tc:ebxml-regrep:xsd:rim:3.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/" xmlns:ows="http://www.opengis.net/ows" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0.2" xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 https://gs-stable.geo-solutions.it/geoserver/schemas/csw/2.0.2/record.xsd">
+  <csw:SearchStatus timestamp="2019-08-13T16:03:43.292Z"/>
```

Comment:
Reuse existing test data

Reply:
in this case i want only one result.

Reply:
so remove the references to gs-stable,geo-solutions.it